src/Utils.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var Matrix = require('ml-matrix');
+
+class Utils {
```

Comment:
Can you make it a usual export of functions instead of a class ?
Then the filename can be lowercase